Locale/pt_BR/translations.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+return array(
+    'Allows users to create and manage their groups by themselves' => 'Permite que os usuários criem e gerenciem seus grupos por conta própria',
+    'Proprietário do grupo' => 'Proprietário do grupo',
```

Comment:
```suggestion
    'Group Owner' => 'Proprietário do grupo',
```

Keys needs to be kept